apps/src/music/player/sequencer/Simple2Sequencer.ts

```diff
@@ -44,6 +46,8 @@ export default class Simple2Sequencer extends Sequencer {
   private startMeasure: number;
   private inTrigger: boolean;
 
+  private currentEventCount: number;
```

Comment:
Instead of a storing a separate index, could we calculate this off of `functionEvents` since each event stores a reference to all of its playback events? Maybe not the most efficient, but could be fine if we're capping at 1000?

Reply:
Update: probably better to use a separate index since we can start with an existing count, resolving this.